clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/appgateway/view/AzureAppGatewayProvider.groovy

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
these classes are needed in order to test the implementation from clouddriver HTTP entry points only (see my PR comment). Basically you can see the AppGateways and do some of the AppGateway related ops via localhost:7002 and curl commands before the switch from using lb to ag.